build/configs/common_download.sh

```diff
@@ -121,6 +121,12 @@ function get_executable_name()
 			else
 				echo "${BOOTPARAM}.bin"
 			fi;;
+		external)
```

Comment:
Should be removed. Right?

Reply:
Oh, this is for `rtk_ext_flash_data`. Then let's rename this. External is toooo common.

Reply:
It is not needed in this file. I removed it.